shell/common/shell_test_external_view_embedder.h

```diff
@@ -15,15 +15,22 @@ namespace flutter {
 ///
 class ShellTestExternalViewEmbedder final : public ExternalViewEmbedder {
  public:
-  using EndFrameCallBack = std::function<void(bool)>;
+  using EndFrameCallBack =
+      std::function<void(bool, fml::RefPtr<fml::RasterThreadMerger>)>;
 
   ShellTestExternalViewEmbedder(const EndFrameCallBack& end_frame_call_back,
-                                PostPrerollResult post_preroll_result)
-      : end_frame_call_back_(end_frame_call_back),
-        post_preroll_result_(post_preroll_result) {}
+                                PostPrerollResult post_preroll_result);
 
   ~ShellTestExternalViewEmbedder() = default;
 
+  // Updates the post preroll result so the |PostPrerollAction| after always
+  // returns the new `post_preroll_result`.
+  void UpdatePostPrerollResult(PostPrerollResult post_preroll_result);
+
+  // Update the post preroll result to PostPrerollResult::kResubmitFrame for
```

Comment:
nit: `Updates`

Reply:
`|PostPrerollResult::kResubmitFrame|` or ``` `PostPrerollResult::kResubmitFrame` ```. I don't know which one is correct here. Maybe @chinmaygarde or @iskakaushik knows

Reply:
done. Updated to `PostPrerollResult::kResubmitFrame`